README.md

```diff
@@ -1,34 +1,159 @@
-This repository contains a README file sample for Daytona Samples and the MIT License.
+# Blogging Application <TS/NextJS>
 
-It can be used as a template to create sample repositories that can be added into [Daytona](https://github.com/daytonaio/daytona).
+## Overview 
+<p>
 
-Once you finish your sample and it gets merged, you can open a PR in the Daytona repo and submit the sample into the [index file](https://github.com/daytonaio/daytona/blob/main/hack/samples/index.json).
+This application leverages MindsDB to deliver intelligent chat assistance. Users can view all posts, like their favorites, and create new posts. The app uses token-based authentication to ensure secure access and interactions. Additionally, users have the ability to report posts, helping to maintain content quality and compliance with community standards.
 
-# Sample <LANGUAGE/FRAMEWORK>
-
-Sample description
+</p>
 
 ---
+## ✨ Features
+- Create a post
+- Report a post
+- It has ai assistance using which we can quickly get quick assistance in articulating our posts.
+- Like a post
+- Segmentation is happening on the backend frontend implementation is yet to be done
+- Follow author
+- Authentication and Authorization is enabled.Users with admin right have privileged rights(this still needs a fix).
+
+## Tech Stack
 
+- **Frontend:** NextJs, Tailwind CSS
+- **Backend:** Next Js
+- **Database:** Postgres DB
+  
 ## 🚀 Getting Started  
 
 ### Open Using Daytona  
 
 1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
 2. **Create the Workspace**:  
    ```bash  
-   daytona create <SAMPLE_REPO_URL> 
+   daytona create https://github.com/Niharika0104/daytona-quest
```

Comment:
As I've already said in the message, please change this so it point to the correct daytonaio sample repo URL